[32mINFO    [0m integration.test_trust:test_trust.py:76 Connection attempt failed: (0)
Reason: Handshake status 500 Internal Server Error -+-+- {'audit-id': '799398bb-9679-44cb-93dc-e8f0e2b6e8f4', 'cache-control': 'no-cache, private', 'content-type': 'application/json', 'warning': '299 - "Use tokens from the TokenRequest API or manually created secret-based tokens instead of auto-generated secret-based tokens."', 'date': 'Fri, 06 Dec 2024 01:29:41 GMT', 'content-length': '169'} -+-+- b'{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"error dialing backend: dial tcp 10.1.0.40:10250: connect: connection refused","code":500}\n'

[32mINFO    [0m integration.test_trust:test_trust.py:78 Retrying (1/20)...
[32mINFO    [0m integration.test_trust:test_trust.py:73 Connection established: <class 'juju.client._definitions.FullStatus'>({'applications': {}, 'branches': {}, 'controller_timestamp': '2024-12-06T01:29:44.426347357Z', 'filesystems': [], 'machines': {}, 'model': <class 'juju.client._definitions.ModelStatusInfo'>({'available_version': '', 'cloud_tag': 'cloud-microk8s', 'meter_status': <class 'juju.client._definitions.MeterStatus'>({'color': '', 'message': '', 'unknown_fields': {}}), 'model_status': <class 'juju.client._definitions.DetailedStatus'>({'data': {}, 'err': None, 'info': '', 'kind': '', 'life': '', 'since': '2024-12-06T01:29:05.58748289Z', 'status': 'available', 'version': '', 'unknown_fields': {}}), 'name': 'test', 'region': 'localhost', 'sla': 'unsupported', 'type_': 'caas', 'version': '3.4.6', 'unknown_fields': {}}), 'offers': {}, 'relations': [], 'remote_applications': {}, 'storage': [], 'volumes': [], 'unknown_fields': {}})